test/e2e/testdata/configs/etcd.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
Is this config valid or invalid? This json format looks outdated (pre-your-enhancement-PR).

Reply:
Yeah that's a leftover unused file. I'll delete it.